includes/core-testing-note-aspnet.md

```diff
@@ -0,0 +1,13 @@
+>[!IMPORTANT]
+>If you're _not_ using the **.NET Core 3.0 SDK**, you must add two additional package references, `Microsoft.AspNetCore.App` and `Microsoft.AspNetCore.Mvc.Testing`.
+>
+>```
+>dotnet add package Microsoft.AspNetCore.App
+>dotnet add package Microsoft.AspNetCore.Mvc.Testing
```

Comment:
This is the first time I've seen this package as a requirement here. Is it always required?

Reply:
According to https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-2.2#test-app-prerequisites it's required.

Reply:
After reading a bit more about this, I believe you're correct. This package is only needed to supplement MVC testing which isn't pertinent to this note.